app/jobs/tweet_stat_adjustment_job.rb

```diff
@@ -0,0 +1,8 @@
+class TweetStatAdjustmentJob < ApplicationJob
+  queue_as :tweet_stat_adjustment
+
+  def perform(article_id)
+    article = Article.find_by(id: article_id)
```

Comment:
the article could be nil, so I suggest adding a `return unless article` if this doesn't exist

Reply:
great catch